HOWTO.md

```diff
@@ -282,6 +285,24 @@ Where:
   - `plain_di`: [OPTIONAL] boolean.
   - `diun`: [OPTIONAL]
     - `mfg_string_type`: device serial number
+    Note: During `DIUN` protocol `mfg_string_type` is passed from `fdo-manufacturing-server` to 
```

Comment:
You need to align this section under `mfg_string_type`, otherwise Markdown shows it at the same level.